src/components/Icons/Sorting/interfaces.ts

```diff
@@ -0,0 +1,15 @@
+import { SVGProps } from 'react';
+
+export interface ISortingColors {
+  activeColor?: string;
+  defaultColor?: string;
+  inactiveColor?: string;
+}
+
+export type SortingOrder = 'asc' | 'desc' | any;
```

Comment:
Do we need the `| any` here?

Reply:
I changed it to be `string | undefined` to match the sorting from `ITable`